spec/git_helper_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'tmpdir'
 require_relative './spec_helper'
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
I'm surprised this is expected, as I am pretty sure I explicitly excluded `*_spec.rb` files from this rule in `.rubocop.yml` already 🤔

Reply:
You are right. Turns out my `develop` wasn't up to date when created this branch. I rebased and it didn't warn me. 

Fixed up in 830d39c.